sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaReadSchemaTransformProvider.java

```diff
@@ -78,7 +77,7 @@ public class KafkaReadSchemaTransformProvider
   public static final TupleTag<Row> OUTPUT_TAG = new TupleTag<Row>() {};
   public static final TupleTag<Row> ERROR_TAG = new TupleTag<Row>() {};
   public static final Schema ERROR_SCHEMA =
-      Schema.builder().addStringField("error").addNullableByteArrayField("row").build();
+      Schema.builder().addNullableByteArrayField("row").build();
```

Comment:
@robertwb Since the input to the ValueMapping DoFn is a `byte[]`, I have to wrap it in a row with an arbitrary field, in this case "row". WDYT about this naming?